src/structure/pdb.jl

```diff
@@ -0,0 +1,273 @@
+the second column. If the `element` property of the `Atom` is set it is used to
+get the element, otherwise the name starts from the second column where
+possible.
+"""
+function spaceatomname(atom::Atom)
+    atom_name = atomname(atom)
+    chars = length(atom_name)
+    @assert chars <= 4 "Atom name is greater than four characters: \"$atom_name\""
+    # In the absence of the element, the first index goes in column two
+    if element(atom) == "" || findfirst(atom_name, element(atom)[1]) == 0
+        cent_ind = 1
+    # The last letter of the element goes in column two where possible
+    else
+        cent_ind = findfirst(atom_name, element(atom)[1]) + length(element(atom)) - 1
+    end
+    @assert cent_ind <= 2 "Atom name is too long to space correctly: \"$atom_name\""
+    if cent_ind == 1 && chars < 4
+        out_string = " $atom_name"
+    else
+        out_string = "$atom_name"
+    end
+    return rpad(out_string, 4)
+end
+
+
+"Form a Protein Data Bank (PDB) format ATOM or HETATM record from an `Atom`."
+pdbline(atom::Atom) = ASCIIString[
+        ishetatom(atom) ? "HETATM" : "ATOM  ",
+        spacestring(serial(atom), 5),
+        " ",
+        spaceatomname(atom),
+        string(altlocid(atom)),
+        spacestring(resname(atom), 3),
+        " ",
+        string(chainid(atom)),
+        spacestring(resnumber(atom), 4),
+        string(inscode(atom)),
+        "   ",
+        # This will throw an error for large coordinate values, e.g. -1000.123
+        spacestring(round(x(atom), 3), 8),
+        spacestring(round(y(atom), 3), 8),
+        spacestring(round(z(atom), 3), 8),
+        spacestring(round(occupancy(atom), 2), 6),
+        # This will throw an error for large temp facs, e.g. 1000.12
+        spacestring(round(tempfac(atom), 2), 6),
+        "          ",
+        spacestring(element(atom), 2),
+        spacestring(charge(atom), 2),
+    ]
+
+
+"""
+Write a `StructuralElementOrList` to a PDB format file. Only ATOM, HETATM, MODEL
+and ENDMDL records are written - there is no header and no TER records.
+"""
+function writepdb(output::IO, element::Union{ProteinStructure, Vector{Model}}, selector_functions::Function...)
+    # If there are multiple models, write out MODEL/ENDMDL lines
+    if length(element) > 1
+        for model in element
+            println(output, "MODEL     ", spacestring(modelnumber(model), 4), repeat(" ", 66))
+            writepdblines(output, model, selector_functions...)
+            println(output, "ENDMDL$(repeat(" ", 74))")
+        end
+    # If there is only one model, do not write out MODEL/ENDMDL lines
+    else
+        writepdblines(output, element, selector_functions...)
+    end
+end
+
+writepdb(output::IO, element::StructuralElementOrList, selector_functions::Function...) = writepdblines(output, element, selector_functions...)
+
+function writepdb(filepath::ASCIIString, element::StructuralElementOrList, selector_functions::Function...)
```

Comment:
This overly-specific type of `filepath` caused [CI failure](https://ci.appveyor.com/project/Ward9250/bio-jl/build/1.0.108/job/j08aoa6obdmxo7y5) on Windows. Of course, you can replace this type with `AbstractString`.